.travis.yml

```diff
@@ -119,5 +119,9 @@ matrix:
       compiler: "native clang++, Job: make check"
       os: osx
 
+    - script:
+      - make typesystem_schema_typescript_tests
```

Comment:
Why not? This wasn't hard. I think there should be an entry point to the repo. For this repo it's the root Makefile as there's no alternative. For example, Node projects have the root package.json as the entry point.